packages/shared/src/components/comments/CommentActionButtons.tsx

```diff
@@ -97,44 +100,51 @@ export default function CommentActionButtons({
 
   return (
     <div className="flex items-center">
-      <QuaternaryButton
-        id={`comment-${comment.id}-upvote-btn`}
-        buttonSize="small"
-        pressed={upvoted}
-        {...getTooltipProps('Upvote')}
-        onClick={toggleUpvote}
-        icon={<UpvoteIcon />}
-        className="btn-tertiary-avocado"
-      >
-        {numUpvotes > 0 ? numUpvotes : null}
-      </QuaternaryButton>
-      <Button
-        buttonSize="small"
-        {...getTooltipProps('Comment')}
-        onClick={() => onComment(comment, parentId)}
-        icon={<CommentIcon />}
-        style={{ marginLeft: sizeN(7) }}
-        className="btn-tertiary-avocado"
-      />
-      <div className="flex-1" />
-      {user?.id === comment.author.id && (
+      <div className="grid grid-cols-4 gap-4">
         <Button
+          id={`comment-${comment.id}-upvote-btn`}
           buttonSize="small"
-          {...getTooltipProps('Edit')}
-          onClick={() => onEdit(comment)}
-          icon={<EditIcon />}
-          className="btn-tertiary mr-2"
+          pressed={upvoted}
+          {...getTooltipProps('Upvote')}
+          onClick={toggleUpvote}
+          icon={<UpvoteIcon />}
+          className="btn-tertiary-avocado"
         />
-      )}
-      {(user?.id === comment.author.id ||
-        user?.roles?.indexOf(Roles.Moderator) > -1) && (
         <Button
           buttonSize="small"
-          {...getTooltipProps('Delete')}
-          onClick={() => onDelete(comment, parentId)}
-          icon={<TrashIcon />}
-          className="btn-tertiary"
+          {...getTooltipProps('Comment')}
+          onClick={() => onComment(comment, parentId)}
+          icon={<CommentIcon />}
+          className="btn-tertiary-avocado"
         />
+        {user?.id === comment.author.id && (
+          <Button
+            buttonSize="small"
+            {...getTooltipProps('Edit')}
+            onClick={() => onEdit(comment)}
+            icon={<EditIcon />}
+            className="btn-tertiary"
+          />
+        )}
+        {(user?.id === comment.author.id ||
+          user?.roles?.indexOf(Roles.Moderator) > -1) && (
+          <Button
+            buttonSize="small"
+            {...getTooltipProps('Delete')}
+            onClick={() => onDelete(comment, parentId)}
+            icon={<TrashIcon />}
+            className="btn-tertiary"
+          />
+        )}
+      </div>
+      {comment.numUpvotes > 0 && (
+        <span
```

Comment:
This should be a tertiary button element. Better in terms of accessibility and design guidelines